docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,122 @@
+How to create a package slice for Chisel
+****************************************
+
+You want to build a chiselled ROCK, but Chisel cannot find the exact bits and pieces of the packages you need to install! This means that said package doesn't yet have the slice definitions you're looking for. In these cases, your best move is to actually **create your own slice definition** (which you can, later on, propose to be merged upstream for everyone else to use).
```

Comment:
Can we use conditional imperatives here https://diataxis.fr/how-to-guides/#the-language-of-how-to-guides ?

If we move to the imperative part, I would remove "You want to build a chiselled ROCK, but Chisel cannot find the exact bits and pieces of the packages you need to install!"

Side comment, can we standarize on not relying on line wrapping? It makes it easier to review and spatially read the source

"which you can, later on, propose to be merged upstream for everyone else to use" should eventually point to a howto (or where upstream is)

Reply:
I have modified the ref publish-slice; it refers to the publish slice PR. Is it good now?

Reply:
@sergiusens the PR looks good to you, or does it need other changes?